src/test/mediapipe/cli/graph.pbtxt

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
```

Comment:
this graph has nothing to do with CLI, does it need to be in CLI directory?

Reply:
is entire cli directory treated as --model_path?

Reply:
if so, i'd suggest naming it the way so noone will use this for other tasks

Reply:
Yes we need specific graph.pbtxt in specific location to test it.